frontend/openchat-agent/src/services/community/candid/idl.js

```diff
@@ -372,6 +372,7 @@ export const idlFactory = ({ IDL }) => {
     'block_index' : BlockIndex,
     'from' : NnsCryptoAccount,
     'memo' : IDL.Nat64,
+    'ledger' : CanisterId,
```

Comment:
With these ledger fields added we won't be able to release the frontend until the backend goes out.
And we can't currently release the backend until we upgrade to candid 0.9.0 which we'll be able to do in a few days.

Reply:
Which is maybe ok... but its something to be aware of.